docs/communicatedds.rst

```diff
@@ -64,12 +63,12 @@ Once we have our configuration file :class:`config.xml` created, we are able to
 Creating new routes
 -------------------
 
-With the knowledge acquired after study and solve this scenario, you should be able to add new connectors between both
+With the knowledge acquired after studying and solving this scenario, it could be possible to add new connectors between both
```

Comment:
I think this section needs explain how to add the new Domain. Because as it is right now, it appears an exercise for the user.

Reply:
I agree